core/bessctl.cc

```diff
@@ -1244,18 +1248,7 @@ class BESSControlImpl final : public BESSControl::Service {
     }
     m2 = it2->second;
 
-    if (is_any_worker_running()) {
-      propagate_active_worker();
-      if (m1->num_active_workers()) {
-        return return_with_error(response, EBUSY, "Module '%s' is in use",
-                                 m1_name);
-      }
-      if (m2->num_active_workers()) {
-        return return_with_error(response, EBUSY, "Module '%s' is in use",
-                                 m2_name);
-      }
-    }
-
```

Comment:
This had a special case: a couple of modules that weren't used by any workers could be connected without any pausing.  Do you think it's worth preserving?

Reply:
Yes, let me look into that a bit more. Indeed one could imagine an application building up a module topology while the workers are running and then making a final connection with minimal worker disruption.